package.json

```
@@ -3,7 +3,8 @@
   "version": "0.0.1",
   "description": "A starting point for the CSS Frameworks CA project",
   "scripts": {
-    "test": "echo \"We will learn more about testing in the Workflow course\" && exit 1"
+    "build": "sass src/scss:dist/css",
+  "watch": "sass --watch src/scss:dist/css & live-server"
```

```suggestion
    "watch": "sass --watch src/scss:dist/css & live-server"
```

Formatting (indendation)